src/generic_ui/locales/en/messages.json

```diff
@@ -1094,14 +1094,13 @@
     "description": "Title on cloud deployment screens, where the user can set up a new server that allows them proxy access.",
     "message": "Create a cloud server"
   },
+  "CREATING_A_CLOUD_SERVER": {
+    "message": "Creating a cloud server"
```

Comment:
We need a description describing, at the very least, where this message is shown.